utils/astxml2markdown.xslt

```diff
@@ -695,7 +698,9 @@ the XML again with the full descriptions, and forms bulleted lists.
 
 <xsl:template match="parameter">
     <xsl:param name="bulletlevel"/>
-    <xsl:value-of select="concat(substring($bulletindent, 0, $bulletlevel * 4 + 1), $bulletchar, ' ')"/>
+    <xsl:text>
+</xsl:text>
+    <xsl:value-of select="concat(substring($bulletindent, 0, $bulletlevel * $tabsize + 1), $bulletchar, ' ')"/>
```

Comment:
What about:

    <xsl:value-of select=“str:padding($bulletlevel * $tabsize + 1)”/>

Reply:
Good call.  Done